tests/distributed/test_pyquest.py

```diff
@@ -0,0 +1,33 @@
+import pyquest
+from pyquest import Register, Circuit
+from pyquest.unitaries import H
+from mpi4py import MPI
+
+
+def main() -> int:
```

Comment:
Do we need to test PyQuEST directly? I think `test_pytketquest.py` achieves the same as this file, but through `pytket-quest`, which is what we want.